python/ray/data/preprocessors/encoder.py

```diff
@@ -242,6 +240,9 @@ class OneHotEncoder(Preprocessor):
             columns will be the same as the input columns. If not None, the length of
             ``output_columns`` must match the length of ``columns``, othwerwise an error
             will be raised.
+        log_capacity: Base 2 logarithm of the maximum size of the internal hash map for
+            top-K calculation. Higher values increase accuracy but use more memory.
+            Defaults to 11 (2048 categories).
```

Comment:
Do we need to expose this as a top-level parameter for now? This parameter couples the interface with a specific implementation, and it might make it harder to change the implementation later